src/http_server.cc

```diff
@@ -29,6 +29,7 @@
 #endif
 
 #include "http_server.h"
+#include "adsbrain_server.h"
```

Comment:
This seems to cause circular dependency. Did you test if the code works after this change?

Reply:
this is .cc file. no problem. Tested.